modules/thermo-cycler/thermo-cycler-arduino/lid.cpp

```diff
@@ -241,19 +247,17 @@ bool Lid::open_cover()
   }
   motor_on();
   bool res;
-  if(move_angle(LID_OPEN_SWITCH_PROBE_ANGLE))  // move down to reach bottom switch
+  // move down a bit to release latch
+  unsigned long push_steps = abs(LID_OPEN_DOWN_MOTION_ANGLE) * STEPS_PER_ANGLE;
+  for (unsigned long i=0;i<push_steps;i++)
   {
-    // lid hit bottom switch
-    solenoid_on();
-    delay(400);
-    move_angle(10);     // move up a bit
-    solenoid_off();
-    res = move_angle(LID_MOTOR_RANGE_DEG);
-  }
-  else
-  {
-    res = false;
+    _motor_step(DIRECTION_DOWN);
   }
+  solenoid_on();
+  delay(400);
+  move_angle(10);     // move up a bit
```

Comment:
Let's make the comments a little more about the why rather than the what - why are we moving up a bit? to clear the solenoid?